### ISSUE TITLE
Trzeba umieć rozróżniać kolory :)

### ISSUE THREAD

**Issue by mraq1234** [mraq1234/Zadanie3#1]
Mam nadzieję, że o to chodziło ...

**Comment by kacgrzes** [comment]
Zgadza się. Brakuje mi tutaj tylko pliku `.gitignore`, który ignorowałby katalog `node_modules/`. Reszta jest jak najbardziej ok

**Comment by kacgrzes** [comment]
nie widzę go w repozytorium. Musisz go dodać, zacommitować i wypchnąć na repo

**Comment by mraq1234** [comment]
U mnie .gitignore jest w pierwszej linii .gitignore :))
Poprawiłem.
